### PR TITLE
Sdit 2269 addresses person inserted process

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonEventListener.kt [hunk 1]
```diff
@@ -66,13 +67,19 @@ class ContactPersonEventListener(
           }
         }
 
-        RETRY_SYNCHRONISATION_PERSON_MAPPING.name -> service.retryCreatePersonMapping(sqsMessage.Message.fromJson())
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonEventListener.kt:69]: move to function so I can convert back to enum

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonMappingApiService.kt [hunk 2]
```diff
@@ -42,39 +42,47 @@ class ContactPersonMappingApiService(@Qualifier("mappingApiWebClient") webClient
     .retrieve()
     .awaitBodyOrNullWhenNotFound()
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonMappingApiService.kt:44]: hmm, ktlint formatting

File: src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonDpsApiMockServer.kt [hunk 3]
```diff
@@ -76,19 +76,18 @@ class ContactPersonDpsApiMockServer : WireMockServer(WIREMOCK_PORT) {
...
-        emailAddresses = request.emailAddresses.map { IdPair(elementType = IdPair.ElementType.EMAIL, nomisId = it.emailAddressId, dpsId = it.emailAddressId * 10) },
-        identities = request.identifiers.map { IdPair(elementType = IdPair.ElementType.IDENTITY, nomisId = it.sequence, dpsId = it.sequence * 10) },
-        employments = request.employments.map { IdPair(elementType = IdPair.ElementType.EMPLOYMENT, nomisId = it.sequence, dpsId = it.sequence * 10) },
-        restrictions = request.restrictions.map { IdPair(elementType = IdPair.ElementType.RESTRICTION, nomisId = it.id, dpsId = it.id * 10) },
-        relationships = request.contacts.map { ContactsAndRestrictions(relationship = IdPair(elementType = IdPair.ElementType.RESTRICTION, nomisId = it.id, dpsId = it.id * 10), restrictions = it.restrictions.map { restriction -> IdPair(elementType = IdPair.ElementType.RESTRICTION, nomisId = restriction.id, dpsId = restriction.id * 10) }) },
-      )
+    fun migrateContactResponse(request: MigrateContactRequest = migrateContactRequest()) = MigrateContactResponse(
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/contactperson/ContactPersonDpsApiMockServer.kt:79]: ktlint format again

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**